examples/minimal/sg_web.tf

```
@@ -1,5 +1,5 @@
 module "sg_vm_web" {
-  source = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  source = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.2.0"
```

Latest release of the security groups module is 1.0.0:
https://github.com/Datatamer/terraform-aws-security-groups/releases
```suggestion
  source = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
```